apps/journeys-admin/src/components/Team/CopyToTeamDialog/CopyToTeamDialog.tsx

```diff
@@ -48,7 +48,7 @@ export function CopyToTeamDialog({
   )
 
   async function handleSubmit(
-    values: FormikValues,
+    values: { teamSelect: string },
     { resetForm }: FormikHelpers<FormikValues>
   ): Promise<void> {
```

Comment:
```suggestion
  async function handleSubmit(values: values: InferType<typeof copyToSchema>, { resetForm }: FormikHelpers<InferType<typeof copyToSchema>>): Promise<void> {
```

```
import { InferType } from 'yup'
```

This will force the schema to stay in sync.